client/src/components/Chart.jsx

```diff
@@ -1,20 +1,20 @@
 /* eslint-disable no-unused-vars */
-
 import React, { useEffect, useRef } from 'react';
 import { useSelector } from 'react-redux';
 import * as chartJs from 'chart.js';
+import PropTypes from 'prop-types';
 
-const Chart = () => {
+const Chart = ({ chartType }) => {
   // Extratring dummy data from the store
-  const dataSample = useSelector(state => state.root.dataSample);
+  const temperatureData = useSelector(state => state.temperatureReducer.temperature);
 
   // Configuring chart parameters and pusing dummy data
   const chartConfig = {
-    type: 'line',
+    type: chartType,
     // passing data to out chart
     data: {
       // labels for X axe, inserting labels data from the store
-      labels: dataSample.labels,
+      labels: temperatureData.labels,
       datasets: [
         {
           // title of the dataset
```

Comment:
I think we can remove these comments as the code is self-explanatory enough 👍

Reply:
Done